pom.xml

```diff
@@ -150,6 +150,7 @@
     <module>jetty-bom</module>
     <module>documentation</module>
     <module>jetty-keystore</module>
+    <module>jetty-unixdomain-server</module>
```

Comment:
this module can be included only within a jdk16 maven profile.

Reply:
It does not require Java 16 to _build_, only to be used, so there is no need for profiles, not in the build nor in the release.

Reply:
in this case you need to remove this https://github.com/eclipse/jetty.project/pull/6522/files#diff-723a454092742efde002c05bbdc642cd7c1a3a4b1d069bffe9ab63fdb1c10bfbR24
```
        <configuration>
          <source>16</source>
          <target>16</target>
          <release>16</release>
        </configuration>
```
because this doesn't build with jdk11.
```
release version 16 not supported
```

Reply:
Ah, good catch!